qa/pull-tester/rpc-tests.py

```diff
@@ -236,7 +236,7 @@ def option_passed(option_without_dashes):
     'compactblocks_2',
     'grapheneblocks',
     'thinblocks',
-    'checkdatasig_activation',
+    Disabled('checkdatasig_activation', "CDSV has been already succesfully activated, keep test around as a template for other OP activation"),
```

Comment:
agreed, I was just following the same convention that we use for a bunch of other tests we disabled in the past. e.g. : 

```python
    Disabled('nov152018_forkactivation','Nov 2018 already activated'),
    Disabled('may152018_forkactivation_1','May 2018 already activated, use it as template to test future upgrade activation'),
    Disabled('may152018_forkactivation_2','May 2018 already activated, use it as template to test future upgrade activation'),
```

I have no problem to delete CDSV test since it is stored in git history in any case.